draft-barnes-sframe-mls.md

```diff
@@ -206,6 +206,40 @@ Epoch 14 +--+-- index=3 --> KID = 0x3e
 MLS also provides an authenticated signing key pair for each participant.  When
 SFrame uses signatures, these are the keys used to generate SFrame signatures.
 
+## Multiple Sender Contexts
+
+Real-time media systems often have strong decoupling between media sources after
+initial setup, including things like providing separate SRTP contexts for
+different media sources.  In SRTP, this is safe because the RTP SSRC is included
+in the nonce computation.  With the system defined here, however, the key and
+nonce only depend on the secret and the sender ID in the KID, so having multiple
+contexts with the same secret and KID leads to nonce reuse.
+
+In order to support this situation, a sender MAY choose to use the high-order
```

Comment:
Since nonce reuse is a serious issue, shouldn't we make a stringer recommendation than `MAY`?

We could say that the default context ID is 0 and sender using more than one context `MUST` indicate the context in high-order bits?